web-api/src/bin/web-api.rs

```diff
@@ -13,16 +13,15 @@
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
 use tracing::instrument;
-use xayn_web_api::{application_names, config, logging, start, Application, WebApi};
+use xayn_web_api::{application_names, logging, start, Application, WebApi};
 
 type Config = <WebApi as Application>::Config;
 
 #[tokio::main]
 #[instrument(err)]
 async fn main() -> Result<(), anyhow::Error> {
-    let config: Config = config::load(application_names!());
-    logging::initialize_global(config.as_ref())?;
-    config.validate()?;
-
+    let config = Config::load(application_names!());
+    logging::initialize_global(config.logging_config())?;
```

Comment:
x3ro: We can't call `logging::initialize_global` in the config loading code as it must _not_ be called when we run integration tests as logging is in that case scoped to the test and it already setup before calling the config through other means